config/webpack.base.config.js

```diff
@@ -2,24 +2,18 @@
 
 const path = require('path')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
+const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin')
 
 const pkg = require(path.resolve(__dirname, '../package.json'))
 
-const build = /production$/.test(process.env.NODE_ENV)
-const mobile = /^mobile/.test(process.env.NODE_ENV)
-const outputFolder = mobile ? 'mobile/www' : 'build'
-const entryFolder = mobile ? 'mobile/src' : 'src'
-
 module.exports = {
-  entry: path.resolve(__dirname, '..', entryFolder, 'main'),
   output: {
-    path: path.resolve(__dirname, '..', outputFolder),
-    filename: build ? 'app.[hash].js' : 'app.js'
+    filename: 'app.js'
   },
   resolve: {
     extensions: ['', '.js', '.jsx', '.json']
   },
-  devtool: build ? '#cheap-module-source-map' : 'eval',
+  devtool: 'cheap-source-map',
```

Comment:
I just try `devtool: 'source-map',` and [it displays](http://webpack.github.io/docs/configuration.html#devtool) the real source code in browser's devtool, very convenient:

![](https://i.imgur.com/vZnOCT3.png)